Firestore/src/AggregateQuery.php

```diff
@@ -36,49 +40,36 @@
  */
 class AggregateQuery
 {
+    use ApiHelperTrait;
     use QueryTrait;
 
-    /**
-     * @var ConnectionInterface
-     * @internal
-     */
-    private $connection;
-
-    /**
-     * @var array
-     */
-    private $query;
-
-    /**
-     * @var string
-     */
-    private $parentName;
-
-    /**
-     * @var array
-     */
-    private $aggregates = [];
+    private FirestoreClient $gapicClient;
+    private array $query;
+    private string $parentName;
+    private array $aggregates = [];
+    private Serializer $serializer;
+    private OptionsValidator $optionsValidator;
 
     /**
      * Create an aggregation query.
      *
-     * @param ConnectionInterface $connection A Connection to Cloud Firestore.
-     *        This object is created by FirestoreClient,
-     *        and should not be instantiated outside of this client.
+     * @param FirestoreClient $gapicClient A FirestoreClient instance.
      * @param string $parent The parent of the query.
      * @param array $query Represents the underlying structured query.
      * @param Aggregate $aggregate Aggregation over the provided query.
      */
     public function __construct(
-        ConnectionInterface $connection,
+        FirestoreClient $gapicClient,
```

Comment:
If possible I'd like to keep the term "Gapic" out of any public interface. I know customers don't actually use this constructor, and also `$firebaseClient` is ambiguous, so this is probably okay, but I wanted to highlight it just in case there's another more appropriate name we can think of.